packages/mocha/src/index.js

```diff
@@ -28,6 +28,37 @@ function convergent(assertion, always) {
   };
 }
 
+/**
+ * Helper to allow automatically running returned objects that implement
```

Comment:
~Helper to~

We should also explain here why we want to do this, and provide some examples of what it enables. I.e. with out it, it means 

- annoying boilerplate in every beforeEach
- having to wrap all of the interaction methods with a call to `run()`